### PR TITLE
[FLINK-22428][docs][table] Translate timezone page into Chinese

### PR DESCRIPTION
**Pull request by hackergin**
## What is the purpose of the change

*Translate timezone page into Chinese*

## Brief change log

  - Translate timezone page into Chinese

## Verifying this change

 - doc change, no need to vierfy 

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 1]
```diff
@@ -105,22 +105,22 @@ Flink SQL> SET table.local-time-zone=America/Los_Angeles;
...
 
-// set to Los_Angeles time zone
+// 设置为Los_Angeles时区
 tEnv.getConfig.setLocalTimeZone(ZoneId.of("America/Los_Angeles"))
 ```
 {{< /tab >}}
 {{< /tabs >}}
 
-The session time zone is useful in Flink SQL, the main usages are:
+session的时区设置在Flink SQL中非常有用, 它的主要用法如下:
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:120]: 我们把 `session` 翻译成 `session (会话)`吧，普通用户好理解点

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 2]
```diff
@@ -249,20 +249,19 @@ Flink SQL> SELECT * FROM MyView3;
...
-Flink SQL defines process time attribute by function `PROCTIME()`, the function return type is `TIMESTAMP_LTZ`.
+### 处理时间和时区
+Flink SQL 使用函数 `PROCTIME()` 来定义处理时间属性, 该函数返回的类型是 `TIMESTAMP_LTZ`.
 
 {{< hint info >}}
-Before Flink 1.13, the function return type of `PROCTIME()` is `TIMESTAMP`, and the return value is the `TIMESTAMP` in UTC time zone,
-e.g. the wall-clock shows `2021-03-01 12:00:00` at Shanghai, however the `PROCTIME()` displays `2021-03-01 04:00:00` which is wrong.
-Flin 1.13 fixes this issue and uses `TIMESTAMP_LTZ` type as return type of `PROCTIME()`, users don't need to deal time zone problems anymore.
+在Flink1.13之前, `PROCTIME()` 函数返回的类型是 `TIMESTAMP`, 返回值是UTC时区下的 `TIMESTAMP`.
+例如: 当上海的时间为 `2021-03-01 12:00:00` 时, `PROCTIME()` 显示的时间却是`2021-03-01 04:00:00`, 因此是错的.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:260]:
```suggestion
例如: 当上海的时间为 `2021-03-01 12:00:00` 时, `PROCTIME()` 显示的时间却是错误的`2021-03-01 04:00:00`.
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 3]
```diff
@@ -249,20 +249,19 @@ Flink SQL> SELECT * FROM MyView3;
...
+这个问题在Flink 1.13中修复了, 因此用户不用再去处理时区的问题了. 
 {{< /hint >}}
 
-The PROCTIME() always represents your local timestamp value, using TIMESTAMP_LTZ type can also support DayLight Saving Time well.
+`PROCTIME()` 返回的时本地时区的时间, 使用 `TIMESTAMP_LTZ` 类型也可以支持夏令时时间.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:264]: minor: 
```suggestion
`PROCTIME()` 返回的是本地时区的时间, 使用 `TIMESTAMP_LTZ` 类型也可以支持夏令时时间.
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 4]
```diff
@@ -368,14 +367,14 @@ Returns the different window start, window end and window proctime compared to c
...
 
 {{< hint info >}}
-Processing time window is non-deterministic, so each run will get different windows and different aggregations. The above example is just for explaining how time zone affects processing time window.
+处理时间窗口是不确定的, 每次运行都会返回不同的窗口和聚合结果. 以上的示例只用于说明时区如何影响处理时间窗口.
 {{< /hint >}}
 
-### Event Time and Time Zone
-Flink supports defining event time attribute on TIMESTAMP column and TIMESTAMP_LTZ column.
+### 事件时间和时区
+Flink支持在 `TIMESTAMP` 列和 `TIMESTAMP_LTZ` 列上定义时间属性.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:374]:
```suggestion
Flink支持在 `TIMESTAMP` 列和 `TIMESTAMP_LTZ` 列上定义事件时间属性.
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 5]
```diff
@@ -368,14 +367,14 @@ Returns the different window start, window end and window proctime compared to c
...
 
-#### Event Time Attribute on TIMESTAMP
-If the timestamp data in the source is represented as year-month-day-hour-minute-second, usually a string value without time-zone information, e.g. `2020-04-15 20:13:40.564`, it's recommended to define the event time attribute as a `TIMESTAMP` column:
+#### TIMESTAMP 上的事件时间属性
+如果source中的时间用于表示年-月-日-小时-分钟-秒, 通常是一个不带时区的字符串, 例如: `2020-04-15 20:13:40.564`. 通常建议在一个 `TIMESTAMP` 列上定义事件时间属性.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:377]:
```suggestion
如果source中的时间用于表示年-月-日-小时-分钟-秒, 通常是一个不带时区的字符串, 例如: `2020-04-15 20:13:40.564`。推荐在 `TIMESTAMP` 列上定义事件时间属性。
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 6]
```diff
@@ -457,8 +456,8 @@ Returns the same window start, window end and window rowtime compared to calcula
 +-------------------------+-------------------------+-------------------------+------+-----------+
 ```
 
-#### Event Time Attribute on TIMESTAMP_LTZ
-If the timestamp data in the source is represented as a epoch time, usually a long value, e.g. `1618989564564`, it's recommended to define event time attribute as a `TIMESTAMP_LTZ` column.  
+#### TIMESTAMP_LTZ 上的事件时间属性
+如果source中的时间为一个epoch时间, 通常是一个long值, 例如: `1618989564564`, 建议将event time属性定义为 `TIMESTAMP_LTZ` 列.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:460]:
```suggestion
如果source中的时间为一个epoch时间, 通常是一个long值， 例如: `1618989564564`， 推荐将事件时间属性定义在 `TIMESTAMP_LTZ` 列上。
```
中文里标点符号，我们都使用中文的吧，文中的可以一起改下

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 7]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
 +-------------------------+-------------------------+-------------------------+------+-----------+
 ```
 
-## Daylight Saving Time Support
-Flink SQL supports defining time attributes on TIMESTAMP_LTZ column, base on this, Flink SQL gracefully uses TIMESTAMP and TIMESTAMP_LTZ type in window processing to support the Daylight Saving Time.
+## 夏令时支持
+Flink SQL支持在 `TIMESTAMP_LTZ`列上定义时间属性, 因此Flink SQL可以在窗口中优雅地使用 `TIMESTAMP` 和 `TIMESTAMP_LTZ` 类型来支持夏令时.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:542]:
```suggestion
Flink SQL支持在 `TIMESTAMP_LTZ`列上定义时间属性， 基于这一特征，Flink SQL 在窗口中使用 `TIMESTAMP` 和 `TIMESTAMP_LTZ` 类型优雅地支持了夏令时。
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 8]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
 long epoch1 = 1615708800000L; // 2021-03-14 00:00:00
 long epoch2 = 1615712400000L; // 2021-03-14 01:00:00
 long epoch3 = 1615716000000L; // 2021-03-14 03:00:00, skip one hour (2021-03-14 02:00:00)
 long epoch4 = 1615719600000L; // 2021-03-14 04:00:00 
 ```
-The tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] will collect 3 hours' data in Los_angele time zone, but it collect 4 hours' data in other non-DST time zones, what user to do is only define time attribute on TIMESTAMP_LTZ column.
+在Los_angele时区下, tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] 将会收集3个小时的数据, 在其他非DST的时区下将会收集4个小时的数据, 用户只需要在 `TIMESTAMP_LTZ` 列上声明时间属性即可.
 
-All windows in Flink like Hop window, Session window, Cumulative window follow this way, and all operations in Flink SQL support TIMESTAMP_LTZ well, thus Flink gracefully supports the Daylight Saving Time zone.   
+Flink的所有窗口(如Hop window, Session window, Cumulative window)都会遵循这种方式, Flink SQL中的所有操作都很好的支持了 `TIMESTAMP_LTZ` ,  因此Flink可以非常优雅的支持夏令时.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:554]: Flink的所有窗口 (如Hop window, Session window, Cumulative window) 都会遵循这种方式，Flink SQL 中的所有操作都很好的支持了 `TIMESTAMP_LTZ` 类型，因此Flink可以非常优雅的支持夏令时。

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 9]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
+以下函数: 
 * LOCALTIME
 * LOCALTIMESTAMP
 * CURRENT_DATE
 * CURRENT_TIME
 * CURRENT_TIMESTAMP
 * NOW()
 
-Flink evaluates their values according to execution mode. They are evaluated for each record in streaming mode. But in batch mode, they are evaluated once as the query starts and uses the same result for every row.
+Flink 会根据执行模式来进行不同计算. 在Streaming模式下会为每条记录都计算出一个值, 但在Batch模式下, 只会在query开始时计算一次, 所有行都使用相同的结果.
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:566]:
```suggestion
Flink 会根据执行模式来进行不同计算， 在 Streaming 模式下这些函数是每条记录都会计算一次，但在 Batch 模式下， 只会在 query 开始时计算一次，所有记录都使用相同的结果。
```

**Review by leonardBang — COMMENTED** [review]
Thanks @hackergin for the timely contribution, I left some comments.

**Comment by hackergin** [comment]
Address @leonardBang  's comments.

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 10]
```diff
@@ -528,7 +527,7 @@ Flink SQL> SET table.local-time-zone=Asia/Shanghai;
 Flink SQL> SELECT * FROM MyView5;
 ```
 
-Returns the different window start, window end and window rowtime compared to calculation in UTC timezone.   
+返回和 UTC 时区下计算时的不同的窗口开始时间， 窗口结束时间和窗口的 rowtime 。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:530]:
```suggestion
相比在 UTC 时区下的计算结果，在 Asia/Shanghai 时区计算的窗口开始时间， 窗口结束时间和窗口的 rowtime 是不同的。
```
相似的这几句，我们这样翻译会好点吗？

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 11]
```diff
@@ -23,29 +23,29 @@ specific language governing permissions and limitations
...
- - `TIMESTAMP_LTZ(p)` is an abbreviation for `TIMESTAMP(p) WITH LOCAL TIME ZONE`, the precision `p` supports range is from 0 to 9, 6 by default.
- - `TIMESTAMP_LTZ` describes an absolute time point on the time-line, it stores a long value representing epoch-milliseconds and an int representing nanosecond-of-millisecond. The epoch time is measured from the standard Java epoch of `1970-01-01T00:00:00Z`. Every datum of `TIMESTAMP_LTZ` type is interpreted in the local time zone configured in the current session for computation and visualization.
- - `TIMESTAMP_LTZ` has no literal representation and thus can not specify from literal, it can derives from a long epoch time(e.g. The long time produced by Java `System.currentTimeMillis()`)
+### TIMESTAMP_LTZ 类型
+ - `TIMESTAMP_LTZ(p)` 是 `TIMESTAMP(p) WITH LOCAL TIME ZONE` 的简写， 精度 `p` 支持的范围是0-9， 默认是6。
+ - `TIMESTAMP_LTZ` 用于描述时间线上的绝对时间点， 使用 long 保存从 epoch 至今的毫秒数， 使用int保存毫秒中的纳秒数。 epoch 时间是从 java 的标准 epoch 时间 `1970-01-01T00:00:00Z` 开始计算。 在计算和可视化时， 每个 `TIMESTAMP_LTZ` 类型的数据都是使用的 session （会话）中配置的时区。
+ - `TIMESTAMP_LTZ` 没有字符串表达形式因此无法通过字符串来指定， 可以通过一个 long 类型的 epoch 时间来转化(例如: 通过Java来产生一个long类型epoch时间 `System.currentTimeMillis()`)
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:48]:
```suggestion
 - `TIMESTAMP_LTZ` 没有字符串表达形式因此无法通过字符串来指定， 可以通过一个 long 类型的 epoch 时间来转化(例如: 通过 Java 来产生一个 long 类型的 epoch 时间 `System.currentTimeMillis()`)
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 12]
```diff
@@ -356,7 +355,7 @@ Flink SQL> SET table.local-time-zone=Asia/Shanghai;
 Flink SQL> SELECT * FROM MyView3;
 ```
 
-Returns the different window start, window end and window proctime compared to calculation in UTC timezone.
+返回和 UTC 时区计算下不同的窗口开始时间， 窗口结束时间和窗口处理时间。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:358]:
```suggestion
相比在 UTC 时区下的计算结果，在 Asia/Shanghai 时区下计算的窗口开始时间， 窗口结束时间和窗口处理时间是不同的。
```
我们把相似的这几句意译一下，更通顺点，你觉得怎么样？

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 13]
```diff
@@ -457,8 +456,8 @@ Returns the same window start, window end and window rowtime compared to calcula
 +-------------------------+-------------------------+-------------------------+------+-----------+
 ```
 
-#### Event Time Attribute on TIMESTAMP_LTZ
-If the timestamp data in the source is represented as a epoch time, usually a long value, e.g. `1618989564564`, it's recommended to define event time attribute as a `TIMESTAMP_LTZ` column.  
+#### TIMESTAMP_LTZ 上的事件时间属性
+如果 source 中的时间为一个 epoch 时间， 通常是一个 long 值， 例如: `1618989564564` ，推荐将事件时间属性定义在 `TIMESTAMP_LTZ` 列上。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:460]:
```suggestion
如果源数据中的时间为一个 epoch 时间， 通常是一个 long 值， 例如: `1618989564564` ，推荐将事件时间属性定义在 `TIMESTAMP_LTZ` 列上。
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md (hunk 10 above)
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:530]: ditto

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 14]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
 
-## Daylight Saving Time Support
-Flink SQL supports defining time attributes on TIMESTAMP_LTZ column, base on this, Flink SQL gracefully uses TIMESTAMP and TIMESTAMP_LTZ type in window processing to support the Daylight Saving Time.
+## 夏令时支持
+Flink SQL支持在 `TIMESTAMP_LTZ`列上定义时间属性， 基于这一特征，Flink SQL 在窗口中使用 `TIMESTAMP` 和 `TIMESTAMP_LTZ` 类型优雅地支持了夏令时。
    
-  
-Flink use timestamp literal to split the window and assigns window to data according to the epoch time of the each row. It means Flink uses `TIMESTAMP` type for window start and window end (e.g. `TUMBLE_START` and `TUMBLE_END`), uses `TIMESTAMP_LTZ` for window time attribute (e.g. `TUMBLE_PROCTIME`, `TUMBLE_ROWTIME`).
-Given a example of tumble window, the DaylightTime in Los_Angele starts at time 2021-03-14 02:00:00:
+Flink 使用时间的字符格式来分割窗口并通过 row 的 epoch 时间来分配窗口。 这意味着 Flink 窗口开始时间和窗口结束时间使用的是 `TIMESTAMP` 类型（例如: `TUMBLE_START` 和 `TUMBLE_END`）， 将 `TIMESTAMP_LTZ` 类型用于窗口的时间属性（例如: `TUMBLE_PROCTIME`， `TUMBLE_ROWTIME`）。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:544]:
```suggestion
Flink 使用时间戳的字符格式来分割窗口并通过每条记录对应的 epoch 时间来分配窗口。 这意味着 Flink 窗口开始时间和窗口结束时间使用的是 `TIMESTAMP` 类型（例如: `TUMBLE_START` 和 `TUMBLE_END`），窗口的时间属性使用的是  `TIMESTAMP_LTZ` 类型（例如: `TUMBLE_PROCTIME`， `TUMBLE_ROWTIME`）。
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 15]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
+给定一个tumble window示例， 在Los_Angele时区下夏令时从 `2021-03-14 02:00:00` 开始：
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:545]:
```suggestion
给定一个 tumble window 示例， 在 Los_Angele 时区下夏令时从 `2021-03-14 02:00:00` 开始：
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 16]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
 ```
 long epoch1 = 1615708800000L; // 2021-03-14 00:00:00
 long epoch2 = 1615712400000L; // 2021-03-14 01:00:00
 long epoch3 = 1615716000000L; // 2021-03-14 03:00:00, skip one hour (2021-03-14 02:00:00)
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:549]:
```suggestion
long epoch3 = 1615716000000L; // 2021-03-14 03:00:00,  手表往前拨一小时，跳过 (2021-03-14 02:00:00)
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 17]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
 long epoch4 = 1615719600000L; // 2021-03-14 04:00:00 
 ```
-The tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] will collect 3 hours' data in Los_angele time zone, but it collect 4 hours' data in other non-DST time zones, what user to do is only define time attribute on TIMESTAMP_LTZ column.
+在 Los_angele 时区下， tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] 将会收集3个小时的数据， 在其他非 DST 的时区下将会收集4个小时的数据，用户只需要在 `TIMESTAMP_LTZ` 列上声明时间属性即可。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:552]:
```suggestion
在 Los_angele 时区下， tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] 将会收集3个小时的数据， 在其他非夏令时的时区下将会收集4个小时的数据，用户只需要在 `TIMESTAMP_LTZ` 列上声明时间属性即可。
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 18]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
 
-All windows in Flink like Hop window, Session window, Cumulative window follow this way, and all operations in Flink SQL support TIMESTAMP_LTZ well, thus Flink gracefully supports the Daylight Saving Time zone.   
+Flink 的所有窗口（如 Hop window， Session window， Cumulative window）都会遵循这种方式， Flink SQL 中的所有操作都很好的支持了 `TIMESTAMP_LTZ` ，因此Flink可以非常优雅的支持夏令时。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:554]:
```suggestion
Flink 的所有窗口（如 Hop window， Session window， Cumulative window）都会遵循这种方式， Flink SQL 中的所有操作都很好地支持了 `TIMESTAMP_LTZ` 类型，因此 Flink 可以非常优雅地支持夏令时。  
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 19]
```diff
@@ -539,35 +538,34 @@ Returns the different window start, window end and window rowtime compared to ca
...
-The tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] will collect 3 hours' data in Los_angele time zone, but it collect 4 hours' data in other non-DST time zones, what user to do is only define time attribute on TIMESTAMP_LTZ column.
+在 Los_angele 时区下， tumble window [2021-03-14 00:00:00,  2021-03-14 00:04:00] 将会收集3个小时的数据， 在其他非 DST 的时区下将会收集4个小时的数据，用户只需要在 `TIMESTAMP_LTZ` 列上声明时间属性即可。
 
-All windows in Flink like Hop window, Session window, Cumulative window follow this way, and all operations in Flink SQL support TIMESTAMP_LTZ well, thus Flink gracefully supports the Daylight Saving Time zone.   
+Flink 的所有窗口（如 Hop window， Session window， Cumulative window）都会遵循这种方式， Flink SQL 中的所有操作都很好的支持了 `TIMESTAMP_LTZ` ，因此Flink可以非常优雅的支持夏令时。  
 
 
-## Difference between Batch and Streaming Mode
-The following time functions:
+## Batch模式和Streaming模式的区别
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:557]:
```suggestion
## Batch 模式和 Streaming 模式的区别
```

File: docs/content.zh/docs/dev/table/concepts/timezone.md [hunk 20]
```diff
@@ -446,7 +445,7 @@ Flink SQL> SET table.local-time-zone=Asia/Shanghai;
 Flink SQL> SELECT * FROM MyView4;
 ```
 
-Returns the same window start, window end and window rowtime compared to calculation in UTC timezone.               
+返回和在 UTC 时区下计算时相同的窗口开始时间， 窗口结束时间和窗口的 rowtime 。
```
Comment (leonardBang) [inline docs/content.zh/docs/dev/table/concepts/timezone.md:448]: ditto

**Review by leonardBang — COMMENTED** [review]
Thanks @hackergin for the update, I only left some final comments.

**Review by leonardBang — APPROVED** [review]
Thanks @hackergin for the great work, LGTM

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**